### PR TITLE
Rename target jar and generate web.config for JavaSE runtime with Schema V2

### PR DESCRIPTION
**Pull request by Flanker32**
Rename target jar and generate web.config for JavaSE runtime with Schema V2, for issue #616

### CODE REVIEW

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java [hunk 1]
```diff
@@ -160,4 +182,44 @@ public void publishWarArtifact(final DeployTarget target, final File warArtifact
...
+    private boolean isJavaSERuntime() throws MojoExecutionException {
+        return runtimeSetting != null &&
+            ((runtimeSetting.getOs().equals("windows") && this.project.getPackaging().equals("jar")) ||
+                (runtimeSetting.getOs().equals("linux") &&
+                    this.runtimeSetting.getLinuxRuntime().stack().equals("JAVA")));
+    }
+
+    private File getProjectJarArtifact(final List<File> artifacts) {
+        final String finalName = project.getBuild().getFinalName();
+        final String fileName = String.format("%s.jar", finalName, project.getPackaging());
```
Comment (andxu) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java]: %s.%s?

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java [hunk 2]
```diff
@@ -160,4 +182,44 @@ public void publishWarArtifact(final DeployTarget target, final File warArtifact
                 String.format("Failed to deploy war file after %d times of retry.", MAX_RETRY_TIMES));
         }
     }
+
+    private boolean isJavaSERuntime() throws MojoExecutionException {
+        return runtimeSetting != null &&
+            ((runtimeSetting.getOs().equals("windows") && this.project.getPackaging().equals("jar")) ||
```
Comment (andxu) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java]: make a util method like `isWindows`, I see a lot of places using this piece of code.

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java [hunk 3]
```diff
@@ -18,17 +18,31 @@
...
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
 public class WebAppUtils {
     public static final String SERVICE_PLAN_NOT_APPLICABLE = "The App Service Plan '%s' is not a %s Plan";
     public static final String CREATE_SERVICE_PLAN = "Creating App Service Plan '%s'...";
     public static final String SERVICE_PLAN_EXIST = "Found existing App Service Plan '%s' in Resource Group '%s'.";
     public static final String SERVICE_PLAN_CREATED = "Successfully created App Service Plan.";
     private static final String CONFIGURATION_NOT_APPLICABLE =
```
Comment (andxu) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java]: be care of the sequence of public static and private static

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java [hunk 4]
```diff
@@ -18,17 +18,31 @@
...
         "The configuration is not applicable for the target Web App (%s). Please correct it in pom.xml.";
+    public static final String JAR_CMD = ":JAR_COMMAND:";
+    public static final String FILENAME = ":FILENAME:";
+    public static final String DEFAULT_JAR_COMMAND = "-Djava.net.preferIPv4Stack=true " +
```
Comment (andxu) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java]: why DEFAULT_JAR_COMMAND  is public?

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java (hunk 1 above)
Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java]: yes, resolved

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java (hunk 2 above)
Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java]: Change to enum to do the comparsion

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java (hunk 3 above)
Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java]: Reorder the statics, public first

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java (hunk 4 above)
Comment (Flanker32) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java]: Fixed it, only set warning messages to public

**Pull request merged**